### PR TITLE
DMP-1646-Darts-Portal-URL

### PR DESCRIPTION
**Pull request by scott-robertson1**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DMP-1590

### Change description ###
a change to the darts portal url to be in line with new business decision where all front facing applications use "apps" in the url instead of "platform" as well as setting up deployments to both the test and demo environments.

### CODE REVIEW

File: apps/darts-modernisation/darts-api/sbox.yaml [hunk 1]
```diff
@@ -10,4 +10,4 @@ spec:
       ingressHost: darts-api.sandbox.platform.hmcts.net
       environment:
         DARTS_GATEWAY_URL: https://darts-gateway.sandbox.platform.hmcts.net
-        DARTS_PORTAL_URL: https://darts-portal.sandbox.platform.hmcts.net
\ No newline at end of file
+        DARTS_PORTAL_URL: https://darts-portal.sandbox.apps.hmcts.net
```
Comment (jackmaloney) [inline apps/darts-modernisation/darts-api/sbox.yaml:13]: Need to remove 'portal'

File: apps/darts-modernisation/darts-portal/sbox.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ spec:
   releaseName: darts-portal
   values:
     nodejs:
-      ingressHost: darts-portal.sandbox.platform.hmcts.net
+      ingressHost: darts-portal.sandbox.apps.hmcts.net
```
Comment (jackmaloney) [inline apps/darts-modernisation/darts-portal/sbox.yaml:10]: Should we remove the word 'portal' from ingress, for consistency?

File: apps/darts-modernisation/darts-portal/stg.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ spec:
   releaseName: darts-portal
   values:
     nodejs:
-      ingressHost: darts-portal.staging.platform.hmcts.net
+      ingressHost: darts-portal.staging.apps.hmcts.net
```
Comment (jackmaloney) [inline apps/darts-modernisation/darts-portal/stg.yaml:10]: Should we remove the word 'portal' from ingress, for consistency?

File: apps/darts-modernisation/darts-portal/test.yaml [hunk 4]
```diff
@@ -7,7 +7,7 @@ spec:
   releaseName: darts-portal
   values:
     nodejs:
-      ingressHost: darts-portal.test.platform.hmcts.net
+      ingressHost: darts-portal.test.apps.hmcts.net
```
Comment (jackmaloney) [inline apps/darts-modernisation/darts-portal/test.yaml:10]: Should we remove the word 'portal' from ingress, for consistency?

**Review by jackmaloney — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Pull request merged**